test/BaselineOutput/Common/OVA/OVA-CV-iris-out.txt

```diff
@@ -1,4 +1,4 @@
-maml.exe CV tr=OVA{p=AvgPer{ lr=0.8 }} threads=- norm=No dout=%Output% data=%Data% seed=1
+maml.exe CV tr=OVA{p=AvgPer{ lr=0.8 }} threads=- norm=No dout=%Output% data=%Data% seed=1 xf=Term{col=Label}
```

Comment:
>xf=Term{col=Label} [](start = 90, length = 18)

Would Covert do it, rather than Term?

Reply:
OVA is multiclass, and for multiclass we would require Key type.

---
In reply to: [263165077](https://github.com/dotnet/machinelearning/pull/2804#discussion_r263165077) [](ancestors = 263165077)